standalone_migrations.gemspec

```diff
@@ -5,13 +5,13 @@
 # stub: standalone_migrations 7.1.0 ruby lib
 
 Gem::Specification.new do |s|
-  s.name = "standalone_migrations".freeze
-  s.version = "7.1.0"
+  s.name = "standalone_migrations_sp".freeze
```

Comment:
did you really wanted to change gem name?

Reply:
@martinskruze no, not change the name, that was a temporary fix to be able to publish the gem. I'll update the PR.

Reply:
Thank you!